schema/datetime_fmt/test_schema.json

```diff
@@ -131,6 +119,28 @@
                "type": "string",
                "description": "shortcut for the time style",
                "enum": ["full", "long", "medium", "short"]
+             },
+             "yearStyle": {
+               "type": "string",
+               "description": "CLDR data for year style",
+               "enum": ["with_era"]
+             },
+             "zoneStyle": {
```

Comment:
Thought: timeZoneName and zoneStyle represent the same thing; maybe collapse them? (Optional)